doc/LocalVMSetup.md

```diff
@@ -16,9 +16,16 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
+## 4. Install GUI so you can setup ssh key
+yum groups install "GNOME Desktop"
+
+startx
+
+## 5. You may need EPEl repo.
 Please refer to: https://fedoraproject.org/wiki/EPEL
 
-## 5. Now you are ready to build and test CORTX. 
+sudo yum install ansible
```

Comment:
What this command does?

Reply:
installs ansible, which is needed in the prerequisites

Reply:
So add a new section here, like "Motr build prerequisites" and add this command under it. Otherwise, it's not clear why it is needed.

Reply:
ok, that makes sense